Automattic/sensei
# Add last activity column to Student Management

Part of https://github.com/Automattic/sensei/issues/4958

### Changes proposed in this Pull Request

Last activity date column is added to the student management table with sorting ability.

Moved the last activity date formating function to utility class because it was being used in multiple places.

### Testing instructions
1. Create a few courses
2. Create a few students
3. Enroll the students to those courses
4. Complete some lessons on different times/days
5. Go the SenseiLMS->Students
6. See if the last activity column is there
7. Try sorting it using the last activity column header
8. See if the similar behavior holds even when filtered by courses

### Screenshot / Video
![Screen Shot 2022-04-06 at 10 48 49 PM](https://user-images.githubusercontent.com/6820724/162026395-62b7d819-2888-4e3a-bd67-8649871fa8e6.png)
